auth-sample/build.gradle

```diff
@@ -67,9 +67,10 @@ android {
 }
 
 dependencies {
-//    implementation files('../auth-lib/build/outputs/aar/auth-release.aar')
-    implementation 'com.spotify.android:auth:2.0.2'
+//    implementation files('../auth-lib/build/outputs/aar/auth-store-debug.aar')
```

Comment:
I would keep this the old way since that is still the default version

Reply:
Yeah makes sense